app/src/main/java/org/wikipedia/main/MainFragment.java

```diff
@@ -480,6 +617,12 @@ private boolean lastPageViewedWithin(int days) {
         return TimeUnit.MILLISECONDS.toDays(System.currentTimeMillis() - Prefs.pageLastShown()) < days;
     }
 
+    private List <String> getLabelList(String imagePath) throws Exception{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['<' is preceded with whitespace.](https://app.codacy.com/app/amawai/android-wikipedia-390/pullRequest?prid=1541123)